opentelemetry-otlp/Cargo.toml

```diff
@@ -15,15 +15,22 @@ keywords = ["opentelemetry", "otlp", "logging", "tracing", "metrics"]
 license = "Apache-2.0"
 edition = "2018"
 build = "build.rs"
+autotests = false
 
 [lib]
 doctest = false
 
+[[test]]
+name = "smoke"
+path = "tests/smoke.rs"
+required-features = ["integration-testing"]
+
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
+async-stream = { version = "0.3.0", optional = true }
```

Comment:
I think you ended up using tokio-stream for #421, is there a reason for not using that here?

Reply:
good point. the TcpListenerStream was only recently added so i missed it. since this PR is supposed to work with tokio 0.2 too i cannot add it yet though. will do once we are on tokio 1

Reply:
i was able to get rid of the dependency since `TcpListener` is a stream on tokio 0.2 - i will use tokio-stream in the tokio v1 PR once this is merged.